### ISSUE TITLE
Can not save filed value to sqlite when contenttype name contain "_"

### ISSUE THREAD

**Issue by everyx** [bolt/bolt#5837]
Details
-------

 - Relevant Bolt Version: [3.1.4]
 - Install type: [ Zip/tarball install ]
 - PHP version: [ 5.5 ]
 - Used web server: [ Nginx ] version [ 1.4.6 ]
 - For UX/UI issues: [ Firefox 49.0.1 ]

Reproduction
------------
​
 1. **Bug summary**: 
    I install bolt and set a contenttype with a field with name "style_no", and everything looks ok but I cannot save the filed data to my sqlite database. when I set a tags taxonomy with name "product_features" and I also can't save the value in a record.

 2. **Config files**:
```yml
product_features:
  name: Features
  slug: product_features
  behaves_like: tags
  postfix: "product features, and split by comma"
  allow_spaces: true
```
```yml
products:
  name: Products
  singular_name: Product
  slug: products
  singular_slug: products
  fields:
    title:
      type: text
      label: "Title"
      required: true
    slug:
      type: slug
      uses: title
    images:
      type: imagelist
      label: "Product images"
      required: true
    styleno:
      type: text
      label: "Style No."
      required: true
    dimensions:
      type: text
      label: "Dimensions"
      info: "Format: WIDTHxHEIGHTxDEPTH in centimeters"
      class: narrow
      variant: inline
      required: true
      pattern: "^[\d.]+x[\d.]+x[\d.]+$"
    brand:
      type: select
      label: "Brand"
      values: brands/name
    gender:
      type: select
      values: [ Men, Women, Boys, Girls, Unisex]
      label: Gender
      required: true
    color:
      type: select
      values: [ Black, Blue, Brown, Gray, Green, Orange, Pink, Purple, Red, Tan, White, Yellow, Other ]
      label: "Color"
    material:
      type: select
      values: [ "Leather", "PU", "PVC", "Fabric", "Other"]
      label: "Material"
    content:
      type: html
      label: "Description"
      options:
        ckeditor:
            height: 500
            autoGrow_maxHeight: 500
            autoGrow_onStartup: false
            uiColor: '#BADA55'
            autoGrow_minHeight: 300
  listing_records: 12
  taxonomy: [ product_features, product_categories]
  default_status: published
  record_template: product_record.twig
  listing_template: product_listing.twig
```

**Comment by GawainLynch** [comment]
@ross assigning this one to you for the time being, as there might be an easy fix to add on to your last workaround.

**Comment by ross** [comment]
> @ross assigning this one to you for the time being, as there might be an easy fix to add on to your last workaround.

I'm probably not the Ross you intended to mention there. :grin: /cc @rossriley

**Comment by GawainLynch** [comment]
> I'm probably not the Ross you intended to mention there.

Hehe … GOLD! 

Well, nice to meet a new Ross then, and hello from the Bolt community :smile:

**Comment by kevinkhill** [comment]
I came here to report this and to my pleasant surprise... already reported and patched! :+1: yay! 

I was beating my head against the wall because I *could not* figure out what was going on. It took a lot of digging on my part before I found out that my "_"s were the culprit.

**Comment by rossriley** [comment]
This shold be finally fixed by #5853
